px-vis-xy-chart.html

```diff
@@ -851,6 +892,39 @@
 
       }, 5);
     },
+    _calcMarkerMargins: function() {
+      if(this.hasUndefinedArguments(arguments)) {
+        return;
+      }
+
+
+      var newMarg;
+
+
+      //use our internal margins
+      newMarg = {
+        'top': this._internalMargin.top,
+        'left': this._internalMargin.left,
+        'right': this._internalMargin.right,
+        'bottom': this._internalMargin.bottom
+      }
+
+      //account for X axis
+      if(this.xAxisLocation === 'bottom') {
+        newMarg.bottom -= 40;
+      } else if(this.xAxisLocation === 'top') {
+        newMarg.top -= 40;
+      }
+
+      //adjust if something is dev set
+      newMarg.top += this.marginMarkersAdjustment.top  ? Number(this.marginMarkersAdjustment.top) : 0;
```

Comment:
Tidy up whitespace here please. This can also probably be shortened to:
```suggestion
      newMarg.top += Number(this.marginMarkersAdjustment.top || 0);
```

Reply:
Done: Updated changes in new commit.